pkg/hooks/launch.go

```diff
@@ -283,6 +284,21 @@ func (h *Hook) processDockerEnv(appCmd, appContainer, appNetwork string, buildDe
 
 			err := h.runApp(appCmd, true)
 			h.logger.Debug("Application stopped with the error", zap.Error(err))
+
+			containerNames, composeErr := getContainerFromComposeFile("kdocker-compose.yaml")
```

Comment:
kdocker compose file is not created everytime, it is created only in the case when keploy-network is not mentioned in the docker compose file and keploy needs the keploy-network to run. So we create a new file called kdocker compose file where the network is keploy-network.

Reply:
ok working on this